aries-site/src/pages/templates/forms.mdx

```diff
@@ -250,3 +252,41 @@ In cases where the user is given an opportunity to express themselves or would l
   href="#sign-in"
   alt="HPE Design System Form where required and optional can be skipped"
 />
+
+## Full-page form guidance
+
+- Use full-page forms for create and edit flows, irrespective of the size of the form.
+- Use a single-column full-page form instead of multi-column layouts.
+- Full-page forms shoud be center-aligned.
+- Use full-page when the form is too long.
+- Use Center modal instead of side drawers/layers for secondary inputs.
```

Comment:
It's not immediately clear what a "secondary input" is.

Also, a "center modal" is a Layer in grommet speak, so this seems misleading. Maybe rewording to "Use a center modal instead of a side drawer".

Reply:
yeah secondary input is tricky trying to think how else to word it it would be a button in the form that takes you to another form... 🤔 

Ill change I agree that wording makes sense

Reply:
changed these